docs/references/api.rst

```diff
@@ -2346,33 +2571,40 @@ This endpoint shows if the machine is attached to a Pro subscription.
 
         .. code-block:: python
 
-           from uaclient.api.u.pro.status.is_attached.v1 import is_attached
-
-           result = is_attached()
+            from uaclient.api.u.pro.services.enable.v1 import enable, EnableOptions
+            result = enable(EnableOptions(service="usg"))
 
       - Expected return object:
 
-        - ``uaclient.api.u.pro.status.is_attached.v1.IsAttachedResult``
+        - ``uaclient.api.u.pro.services.enable.v1.EnableResult``
 
           .. list-table::
-             :header-rows: 1
+            :header-rows: 1
+
+            * - Field Name
+              - Type
+              - Description
+            * - ``enabled``
+              - ``List[str]``
+              - List of services that were enabled.
+            * - ``disabled``
+              - ``List[str]``
+              - List of services that were disabled
```

Comment:
For consistency of appearance, I'd suggest putting full stops at the end of each of these descriptors (since the last one needs one, they should all have one)

Reply:
Ah yep I missed it - thanks!